assets/pmgbootstrap/_theme.scss

```diff
@@ -15,25 +17,156 @@
+        padding-left: 0;
+        padding-right: 0;
+        margin-bottom: 20px;
+        @media (min-width: $screen-sm-min) {
+            width: $container-sm;
+        }
+        @media (min-width: $screen-md-min) {
+            width: $container-md;
+        }
+        @media (min-width: $screen-lg-min) {
+            width: $container-lg;
+        }
+    }
+
+}
+
 html, body {
     font-weight: 300;
     background-color: $gray-lighter;
 }
 
+
 body {
-    & > .container {
-        @include main-container();
+    .navbar {
+        padding: 10px;
+    }
+    .main-content, .main, .main-wrap {
+        .title-bar .breadcrumb {
+            background-color: transparent;
+            text-transform: lowercase;
+            padding-left: 0px;
+        }
     }
-    & > .container-fluid {
-        @include main-container-fluid();
+}
+
+.navbar-brand .logo {
+    display: inline-block;
+    width: 20px;
+    height: 16px;
+    background-repeat: no-repeat;
+    position: relative;
+    top: 3px;
+    background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxOS4xLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9Ii0xNzggMzY0LjYgNzkuNCA2NC40IiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IC0xNzggMzY0LjYgNzkuNCA2NC40OyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+DQo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPg0KCS5zdDB7ZmlsbDojMDBBMkI5O30NCgkuc3Qxe2ZpbGw6I0ZGRkZGRjt9DQo8L3N0eWxlPg0KPGc+DQoJPHBhdGggY2xhc3M9InN0MCIgZD0iTS0xMTkuNSwzODcuOWwtMjEuNS0xOC44Yy00LjYtNC0xMS41LTMuNS0xNS41LDFsLTE4LjgsMjEuNWMtNCw0LjYtMy41LDExLjUsMSwxNS41bDIxLjUsMTguOA0KCQljMy40LDMsOC4xLDMuNSwxMS45LDEuN2wtOC4zLTcuM2MwLDAsMCwwLTAuMS0wLjFsLTE5LjgtMTcuM2MtMi4zLTItMi41LTUuNS0wLjUtNy44bDE3LjQtMTkuOGMyLTIuMyw1LjUtMi41LDcuNy0wLjVsMTkuOCwxNy40DQoJCWMyLjMsMiwyLjUsNS41LDAuNSw3LjdsLTMuOCw0LjRsNSw0LjNsNC42LTUuMkMtMTE0LjUsMzk4LjgtMTE0LjksMzkxLjktMTE5LjUsMzg3LjkiLz4NCgk8cGF0aCBjbGFzcz0ic3QxIiBkPSJNLTEwMi4zLDM4OC4xbC0yMS41LTE4LjhjLTMuNC0zLTguMS0zLjUtMTEuOS0xLjdsOC4zLDcuM2MwLDAsMCwwLDAuMSwwbDE5LjgsMTcuNGMyLjMsMiwyLjUsNS41LDAuNSw3LjcNCgkJbC0xNy4zLDE5LjljLTIsMi4zLTUuNSwyLjUtNy43LDAuNWwtMTkuOS0xNy40Yy0yLjMtMi0yLjUtNS41LTAuNS03LjdsMy45LTQuNGwtNS00LjNsLTQuNiw1LjJjLTQsNC42LTMuNSwxMS41LDEsMTUuNWwyMS41LDE4LjgNCgkJYzQuNiw0LDExLjUsMy41LDE1LjUtMWwxOC44LTIxLjVDLTk3LjMsMzk5LTk3LjcsMzkyLjEtMTAyLjMsMzg4LjEiLz4NCjwvZz4NCjwvc3ZnPg0K');
+
+    img {
+        display: none;
     }
+}
+
+html:not(.sidenav) body {
+    @include topbar-not-side-nav;
+}
+
+@media (max-width: $sidenavbar-min-width) {
+    @include topbar-not-side-nav;
+}
+
 
-    .main, .main-wrap {
-        & > .container {
-            @include main-container();
+@media (min-width: $sidenavbar-min-width) {
+    html.sidenav body > .body {
+        display: flex;
+        min-height: 100vh;
```

Comment:
Do people use DWH on anything other than desktop? Some phones/tablets had issues with 100vh (not sure if it's still an issue)